pkg/proc/bininfo.go

```diff
@@ -296,6 +288,12 @@ func loadBinaryInfo(bi *BinaryInfo, image *Image, path string, entryPoint uint64
 	return errors.New("unsupported operating system")
 }
 
+// SetLogger sets the internal logger to use for
+// the BinaryInfo object. Mostly useful for tests.
+func (bi *BinaryInfo) SetLogger(l *logrus.Entry) {
```

Comment:
It looks like this is only used by a single test, isn't there any way around this?

Reply:
Hopefully, I can look into it, I'd rather remove it as well.